src/Package/MSTest/MSTest.nuspec

```diff
@@ -6,13 +6,13 @@
       <group targetFramework="netstandard2.0">
         <dependency id="MSTest.TestAdapter" version="[$version$]" />
         <dependency id="MSTest.TestFramework" version="[$version$]" />
-        <dependency id="Microsoft.NET.Test.Sdk" version="[$TestPlatformVersion$]" />
+        <dependency id="Microsoft.NET.Test.Sdk" version="$TestPlatformVersion$" />
```

Comment:
Can it be $TestPlatformVersion$ or higher?

Reply:
This used to be = version (syntax is version between brackets), this is now >=.